packages/root-config/src/bootstrap/autocomplete/autocomplete.js

```diff
@@ -6,7 +6,7 @@ import {ServiceProvider, AutocompleteService, AutocompleteContextService, Reposi
  * If there is no selected repository, the request will not be made.
  */
 const isAutocompleteEnabled = () => {
-  const currentRepository = ServiceProvider.get(RepositoryStorageService).get('selecterRepositoryId').getValue();
+  const currentRepository = ServiceProvider.get(RepositoryStorageService).get('selectedRepositoryId').getValue();
```

Comment:
Please change to `RepositoryContextService.SELECTED_REPOSITORY_ID` to mitigate further errors like this

Reply:
I see this pattern used in other places